coverage/coverage-final.json

```diff
@@ -0,0 +1,3 @@
+{"/Users/Deng/chun/sei/sprints/sycamore/client/src/components/RatingsAndReviews/ProductBreakdown/ProductBreakdown.jsx": {"path":"/Users/Deng/chun/sei/sprints/sycamore/client/src/components/RatingsAndReviews/ProductBreakdown/ProductBreakdown.jsx","all":false,"statementMap":{"0":{"start":{"line":1,"column":0},"end":{"line":1,"column":26}},"1":{"start":{"line":2,"column":0},"end":{"line":2,"column":33}},"2":{"start":{"line":3,"column":0},"end":{"line":3,"column":0}},"3":{"start":{"line":4,"column":0},"end":{"line":4,"column":32}},"4":{"start":{"line":5,"column":0},"end":{"line":5,"column":10}},"5":{"start":{"line":6,"column":0},"end":{"line":6,"column":27}},"6":{"start":{"line":7,"column":0},"end":{"line":7,"column":4}},"7":{"start":{"line":8,"column":0},"end":{"line":8,"column":2}},"8":{"start":{"line":9,"column":0},"end":{"line":9,"column":0}},"9":{"start":{"line":10,"column":0},"end":{"line":10,"column":32}}},"s":{"0":1,"1":1,"2":1,"3":1,"4":1,"5":1,"6":1,"7":1,"8":1,"9":1},"branchMap":{"0":{"type":"branch","line":4,"loc":{"start":{"line":4,"column":25},"end":{"line":8,"column":2}},"locations":[{"start":{"line":4,"column":25},"end":{"line":8,"column":2}}]}},"b":{"0":[1]},"fnMap":{"0":{"name":"ProductBreakdown","decl":{"start":{"line":4,"column":25},"end":{"line":8,"column":2}},"loc":{"start":{"line":4,"column":25},"end":{"line":8,"column":2}},"line":4}},"f":{"0":1}}
```

Comment:
What is this coverage-final for?

Reply:
that's folder created by one of the test library. I removed the entire folder.